projects/storefrontlib/shared/components/media/media.component.ts

```diff
@@ -20,6 +20,25 @@ import { ImageLoadingStrategy, Media, MediaContainer } from './media.model';
 import { MediaService } from './media.service';
 import { USE_LEGACY_MEDIA_COMPONENT } from './media.token';
 
+/**
+ * The HTML element rendered in the template can be either `<img>` or `<picture>`,
+ * depending on the input value of `elementType`, which defaults to `'img'`.
+ *
+ * In the case of a `<picture>` element, the `srcset` attribute will always include only one URL.
+ *
+ * For example, instead of:
+ *
+ * ```html
+ * <source media="(min-width: 960px)" srcset="image-1400.jpg 1x, image-2800.jpg 2x">
+ * ```
+ *
+ * Use additional formats and additional media queries, such as:
+ *
+ * ```html
+ * <source media="(min-width: 960px)" srcset="image-1400.jpg">
+ * <source media="(min-width: 960px) and (-webkit-min-device-pixel-ratio: 2)" srcset="image-2800.jpg">
```

Comment:
Please help me understand, shouldn't the <source> tags be swapped? I mean, shouldn't the more specific one be the first?
Currently, I understand that the first `<source>` will be used in all cases (regardless of pixel density), because the constraint is too general.
Please correct me if I'm wrong.

Reply:
Yes, you have right. The more specific should be first